### PR TITLE
chore: mqtt initial skeleton

### PR DESCRIPTION
**Pull request by arthurborgesdev**
In this PR, two commits were included with two main additions. The first is an initial skeleton with MQTT pub/sub in the same file for testing purposes with the async approach (async-mqtt). Then, after tests succeeded, the mqtt part was encapsulated in a separate file called "iot".

### CODE REVIEW

File: src/mqtt.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      Logger.debug(`Something wrong with MQTT start: ${e}`);
+    }
+  };
+
+  client.on("connect", initMqtt);
+
+  const handleMqttMessage = async (topic, message) => {
+    // message is Buffer
+    Logger.debug(message.toString());
+    await client.end();
```
Comment (luk3skyw4lker) [inline src/mqtt.ts]: Does this client.end here kills the connection with the MQTT server? If so, we would be able to only receive the first message sent by the meter, it doesn't make a lot of sense for me

Comment (arthurborgesdev) [reply src/mqtt.ts]: Yes, it ends it. Despite being only a demo, it makes sense to decide the logic here, right now in the beginning. Should we close this connection on a node process.exit, for example?

Comment (luk3skyw4lker) [reply src/mqtt.ts]: @arthurborgesdev Actually the connection shouldn't be closed at all, you should watch the topic all the time for new messages and everytime one comes save it at the DB or else we would have to restart the app (or the connection itself) everytimes a message comes

Comment (arthurborgesdev) [reply src/mqtt.ts]: Acknowledged! Thanks for the explanation!

**Review by luk3skyw4lker — APPROVED** [review]
The connection and test message are all working fine!

**Pull request merged**